### PR TITLE
add 66 plus-one

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 20m

### CODE REVIEW

**Comment by kobakobashu** [comment]
Please check my code.

File: 66.plus-one.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# @lc code=start
+class Solution:
+    def plusOne(self, digits: List[int]) -> List[int]:
+        """
+        time: O(len(digits))
+        space: O(len(digits))
+        """
+        number_str = ""
+        for digit in digits:
+            number_str += str(digit)
```
Comment (ryuji0123) [inline 66.plus-one.py:16]: use list to avoid str concatination.

**Review by ryuji0123 — CHANGES_REQUESTED** [review]
You can write simply

```py
class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        return list(str(int(''.join([str(d) for d in digits])) + 1))
```

**Comment by kobakobashu** [comment]
I fixed my code.  Please check it.

**Review by ryuji0123 — APPROVED** [review]
LGTM

**Pull request merged**